### PR TITLE
Ensure mage's task run with GO from .GO_VERSION

### PR DESCRIPTION
**Pull request by ph**
## What does this PR do?

Ensure mage's task run with GO from .GO_VERSION

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by ph** [comment]
@mergifyio update

**Review by michalpristas — APPROVED** [review]

**Comment by ph** [comment]
@mergifyio update

**Comment by ph** [comment]
@Mergifyio update

**Comment by ph** [comment]
@Mergifyio update

No error in build, will try one more time and investigate the CI.

**Comment by v1v** [comment]
This is now solved, so I'll close this now

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -88,7 +89,7 @@ pipeline {
               withGithubNotify(context: "Build-${PLATFORM}") {
                 deleteDir()
                 unstashV2(name: 'source', bucket: "${JOB_GCS_BUCKET}", credentialsId: "${JOB_GCS_CREDENTIALS}")
-                withMageEnv(){
+         j      withMageEnv(version: "${env.GO_VERSION}"){
```
Comment (v1v) [inline .ci/Jenkinsfile:92]: typo

**Pull request closed without merging**